### PR TITLE
Node Electron Integration & Eslint

### PR DESCRIPTION
**Pull request by neoJINXD**
I fixed the divergence between the web app and the electron app.
Now, both versions use the same files (in the public folder) to load the app in the view.

Also, I added Eslint for checking the linting on the JavaScript side, using the Airbnb config (I chose this since it shows that it is the more popular js code standard when checking github).

### CODE REVIEW

**Comment by VisusAdAstra** [comment]
why do we have conflict file, you mean we will remove it in staging version?
public/serverScripts.js

File: index.js [hunk 1]
```diff
@@ -1,45 +1,35 @@
...
+  const input = req.body.expression;
+  // console.log('my inpit: ', input);
+
+  const option = {
+    args: [input],
+  };
+
+  // TODO change to always listening python shell instead of creating and deleting everytime
+  /* eslint-disable-next-line */
+  const pyshell = new python('calculate.py', option);
```
Comment (VisusAdAstra) [inline index.js:26]: just to make sure, are you sure this shell isn't killed everytime?

File: main.js [hunk 2]
```diff
@@ -1,46 +1,27 @@
...
-      height: 600
-    })
-    //window.loadURL('https://youtube.com')
-    //console.log(`Loading: file://${__dirname}/public/index.html`);
-    
-    window.loadURL(`file://${__dirname}/index.html`)
-    window.setAlwaysOnTop(true)
+function createWindow() {
+  const window = new BrowserWindow({
+    width: 800,
```
Comment (VisusAdAstra) [inline main.js:11]: what is the unit here?

Comment (neoJINXD) [reply main.js:11]: this is the default size of the window, I just kept it as the default given by the Electronjs docs for now

File: index.js (hunk 1 above)
Comment (neoJINXD) [reply index.js:26]: I am working on this on a seperate branch and will make a seperate PR once that's ready to not overload and put too much on this PR

**Comment by neoJINXD** [comment]
File conflict seemed to have been from me changing the serverScripts to fit with the eslint standards I setup, got that fixed

**Comment by neoJINXD** [comment]
Also, will see if I can add a check to our CI with github actions to check the javascript linting too

**Pull request merged**